nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -903,15 +903,13 @@ def permuted_ols(
             )
 
             if two_sided_test:
-                # Label positive and negative clusters separately
-                n_positive_clusters = np.max(labeled_arr3d)
+                # Add negative cluster labels
                 temp_labeled_arr3d, _ = label(
                     scores_original_data_3d < -threshold_t,
                     bin_struct,
                 )
-                temp_labeled_arr3d[
-                    temp_labeled_arr3d > threshold_t
```

Comment:
Would the bug be fixed by changing `threshold_t` here to `0`?

Reply:
Yes, you are right. I did not fully understand the logic at first and reversed the order it seems: 

`temp_labeled_arr3d > 0` results in `np.concatenate([positive cluster labels], [negative cluster labels])`

And my change results in `np.concatenate([negative clusters labels], [positive cluster labels])`

Which do you prefer?

Reply:
I think the diff would be cleaner if you just changed the threshold, but I don't feel too strongly about it. I don't know if changing the order of clusters in the labeled map (negative before positive, instead of the opposite) is something users will notice, so it's up to you.